sokol_app.h

```diff
@@ -928,27 +928,49 @@ _SOKOL_PRIVATE void _sapp_frame(void) {
 
 /*== MacOS ===================================================================*/
 #if defined(TARGET_OS_IPHONE) && !TARGET_OS_IPHONE
+
+#if defined(SOKOL_METAL)
 #import <Metal/Metal.h>
 #import <MetalKit/MetalKit.h>
+#elif defined(SOKOL_GLCORE33)
+#include <Cocoa/Cocoa.h>
+#include <OpenGL/gl3.h>
+#endif
 
 @interface _sapp_macos_app_delegate : NSObject<NSApplicationDelegate>
 @end
 @interface _sapp_macos_window_delegate : NSObject<NSWindowDelegate>
 @end
+#if defined(SOKOL_METAL)
 @interface _sapp_macos_mtk_view_dlg : NSObject<MTKViewDelegate>
 @end
 @interface _sapp_macos_view : MTKView
 {
     NSTrackingArea* trackingArea;
 }
 @end
+#elif defined(SOKOL_GLCORE33)
+@interface _sapp_macos_view : NSOpenGLView
+{
+    NSTrackingArea* trackingArea;
+}
+- (void)timerFired:(id)sender;
+- (void)prepareOpenGL;
+- (void)drawRect:(NSRect)bounds;
```

Comment:
It seems like these function definitions are not needed here, it compiles without them. I don't know Obj-C but it feels better to be explicit here and keep them. Or is it related to what functions are public or private?

Reply:
These are part of the interface definition for _sapp_macos_view, with the implementation further down below. I agree, I think it's better to keep them there, when reading the code top-to-bottom it's clearer which methods from the NSOpenGLView parent class _sapp_macos_view overrides.

Reply:
Ok!